OpenMEEG/cmake/UseOpenBLAS.cmake

```diff
@@ -26,6 +26,16 @@ if (USE_OPENBLAS)
     else()
         find_package(OpenBLAS ${FIND_MODE} MODULE)
     endif()
+    if (APPLE)
+            set(LIBS)
+            foreach(lib ${OpenBLAS_LIBRARIES})
+                get_filename_component(reallib ${lib} REALPATH)
+                list(APPEND LIBS ${reallib})
+            endforeach()
+            install(FILES ${LIBS} DESTINATION lib
+                PERMISSIONS OWNER_EXECUTE OWNER_WRITE OWNER_READ
+                GROUP_EXECUTE GROUP_READ)
```

Comment:
of course. I have the binaries on the ci, and I was testing it.
I've extracted it locally, and did:
'export DYLD_LIBRARY_PATH=../lib'
so that I run without problems ./om_assemble
Does it means it is ok ? or do I need to uninstall openblas to be sure that it is using the lib I provided in the ../lib, and not the system lib ?